package.json

```diff
@@ -369,18 +369,18 @@
 		"test": "gulp test"
 	},
 	"devDependencies": {
-		"@types/fs-extra": "^9.0.11",
-		"@types/glob": "^7.1.3",
-		"@types/gulp": "^4.0.8",
+		"@types/fs-extra": "^9.0.12",
+		"@types/glob": "^7.1.4",
+		"@types/gulp": "^4.0.9",
 		"@types/gulp-sourcemaps": "^0.0.34",
-		"@types/js-yaml": "^4.0.1",
-		"@types/mocha": "^8.2.2",
-		"@types/node": "^14.16.0",
-		"@types/node-netstat": "^1.8.0",
+		"@types/js-yaml": "^4.0.2",
+		"@types/mocha": "^8.2.3",
+		"@types/node": "^14.17.5",
```

Comment:
I usually try to match the version of Node used by VS Code (i.e. Electron), to ensure we don't try to use runtime features that don't exist in older versions (and Electron always trails the latest Node.js versions).

Reply:
Thanks for pointing out this. I just used `npm update` and didn't know of these gotchas!